pkg/registry/apis/folders/conversions.go

```diff
@@ -201,23 +199,6 @@ func setParentUID(u *unstructured.Unstructured, parentUid string) error {
 	return nil
 }
 
-func getLegacyID(meta utils.GrafanaMetaAccessor) (int64, error) {
-	var i int64
-
-	repo, err := meta.GetRepositoryInfo()
-	if err != nil {
-		return i, err
-	}
-
-	if repo != nil && repo.Name == "SQL" {
```

Comment:
What are some other possible repo name values?

Reply:
There are lots :) for file based provisioning, it will be the name of the provisioner. For plugin imported dashboards / folders, it will be "plugin". For git sync, it'll be the repo